synapse/lib/stormtypes.py

```diff
@@ -2091,6 +2090,9 @@ async def wget(self, url, headers=None, params=None, method='GET', json=None, bo
         timeout = await toprim(timeout)
         proxy = await toprim(proxy)
 
+        if proxy is not None:
+            self.runt.confirm(('storm', 'lib', 'axon', 'proxy'))
```

Comment:
I thought we were moving this to `storm.lib.inet.http.proxy` as a single permission for controlling proxy bypass?

Reply:
I must've missed that part of the convo.